### PR TITLE
Exact pruning algorithm

### PR DESCRIPTION
**Pull request by frapac**
Add function `exactpruning!` to remove cuts with an exact test

Source: 
https://hal.inria.fr/hal-00753578/document

### CODE REVIEW

File: src/exact.jl [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    red
+end
+
+"""State whether a cut is dominated with a tolerance epsilon."""
+function isdominated(A, b, islb, isfun, k, solver, lb, ub, epsilon)
+    # we use MathProgBase to solve the test
+    # For instance, if islb & isfun, the LP becomes:
+    # min - y
+    #     x ∈ R^n, y ∈ R
+    #     y <= (A[k, :] - A[j, :])*x + b[k] - b[j]     ∀ i != k
```
Comment (blegat) [inline src/exact.jl]: \forall i -> \forall j

**Review by blegat — APPROVED** [review]
Looks good ! You need to add MathProgBase in the REQUIRE file though :)

**Comment by frapac** [comment]
You are right! I completely forgot to update `REQUIRE` ...

**Comment by blegat** [comment]
I have addressed the typo in the comment and added MPB in the REQUIRE file. Let's see what Travis has to tell us now :)

**Comment by frapac** [comment]
Thanks!

**Comment by blegat** [comment]
@frapac One line is not covered in `exact.jl` :(

**Comment by frapac** [comment]
@blegat Ok, I will fix this!

**Pull request merged**